js/scripts.js

```diff
@@ -0,0 +1,113 @@
+    //New game
+
+var newGameBtn = document.getElementById("js-new-game");
+
+document.getElementById("game").style.display = "none";
+document.getElementById("js-new-game-section" && "winner-section").style.display = "none";
+
+function setNewGame() {
+    document.getElementById("game").style.display = "block";
+    document.getElementById("js-new-game-section").style.display = "none";
+    var playerN = prompt("Please enter your name", "Player");
+        if ((typeof(playerN) == "string") && (playerN != "")){
+                playerName.innerHTML = playerN;
+        };
+
+    var plPoints = 0,
+        comPoints = 0;
+
+        playerPoints.innerHTML = plPoints;
+        computerPoints.innerHTML = comPoints;
+        playerWinner.innerHTML = "";
+        computerWinner.innerHTML = "";
+};
+
+newGameBtn.addEventListener("click", setNewGame);
+
+    //Computer
+
+function getComputerPick() {
+    var possiblePicks = ['Rock', 'Paper', 'Scissors'];
+        if (possiblePicks[Math.floor(Math.random()*3)] == "Rock") {
+                computerPick = "Rock"
+        } else if (possiblePicks[Math.floor(Math.random()*3)] == "Paper") {
+                computerPick = "Paper";
+        } else if (possiblePicks[Math.floor(Math.random()*3)] == "Scissors") {
+                computerPick = "Scissors";     
+        };
+        computerChoice.innerHTML = computerPick;
+};
+
+    //Player
+
+var rock = document.getElementById('js-rock'),
+    paper = document.getElementById('js-paper'),
+    scissors = document.getElementById('js-scissors'),
+    playerPick = 0,
+    computerPick = 0;
+
+    rock.addEventListener('click', plPickRock);
+    paper.addEventListener('click', plPickPaper);
+    scissors.addEventListener('click', plPickScissors);
+
+function plPickRock() {
```

Comment:
write one fuction for `plPickRock`, `plPickPaper` and `plPickScissors` where playerPick variable will be taken from arguments